client/src/components/GameEnv/2DEnv/2DEnv.jsx

```diff
@@ -1,21 +1,60 @@
-import React from 'react';
+/* eslint-disable array-callback-return */
```

Comment:
we should not be using eslint-disable, find a workaround to get your code working and to not bypass the linter